PicoProducer/python/analysis/ModuleTauPair.py

```diff
@@ -39,9 +41,9 @@ def __init__(self, fname, **kwargs):
     self.year       = kwargs.get('year',     2022           ) # integer, e.g. 2017, 2018
     self.era        = kwargs.get('era',      '2022postEE'   ) # string, e.g. '2017', 'UL2017'
     self.ees        = kwargs.get('ees',      1.0            ) # electron energy scale
-    self.tes        = kwargs.get('tes',      None           ) # tau energy scale; if None, recommended values are applied
```

Comment:
@jmalvaso @pmastrap In general it's better to not change these default settings if it's for your own personal reasons, because it might break other people's setups. If you want to disable this, you need to do so via your sample list or the way you define the channel. (Anyway, I'll try to make this easier / more clear in the new configuration paradigm.)

Reply:
@IzaakWN  Should I restore these setting to the default value and create a new pull request or, for this time, it is fine like this?

Reply:
I would suggest to just add a commit that undoes changing these defaults if that's okay with you?

Reply:
Yes, it's fine. I will do it asap

Reply:
@IzaakWN I think I  added the commit in the proper way, but please let me know if it's ok now